core/beacon/errors.go

```diff
@@ -41,9 +41,9 @@ var (
 	INVALIDBLOCKHASH     = "INVALID_BLOCK_HASH"
 	INVALIDTERMINALBLOCK = "INVALID_TERMINAL_BLOCK"
 
-	GenericServerError = rpc.CustomError{Code: -32000, ValidationError: "Server error"}
-	UnknownPayload     = rpc.CustomError{Code: -32001, ValidationError: "Unknown payload"}
-	InvalidTB          = rpc.CustomError{Code: -32002, ValidationError: "Invalid terminal block"}
+	GenericServerError       = rpc.CustomError{Code: -32000, ValidationError: "Server error"}
```

Comment:
Just general reminder, it looks like the error definition is updated in spec https://github.com/ethereum/execution-apis/blob/main/src/engine/specification.md#authentication